docs/snippets/package/aws-with-vpc/definition.yaml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
All these `definition.yaml` files are identical; I just copied the existing one from `docs/snippets/compose`.